test/lib/ansible_test/_data/sanity/import/importer.py

```diff
@@ -532,6 +532,7 @@ def capture_output(capture):
                 )
 
                 # Temporary solution until there is a vendored copy of distutils.version in module_utils.
+                # Some of our dependencies such as packaging.tags also import distutils, which we have no control over
                 # The warning text is: The distutils package is deprecated and slated for removal in Python 3.12.
                 # Use setuptools or check PEP 632 for potential alternatives
                 warnings.filterwarnings(
```

Comment:
> We may need to wait even longer, since there are many deps we don't control that could surface this warning in collections as well.

I was going to look at our other deps to see if they have similar problems. Maybe we'll get lucky.